contracts/payloads/IGP95/PayloadIGP95.sol

```diff
@@ -0,0 +1,973 @@
+
+                VAULT_FACTORY.setVaultAuth(
+                    XAUT_GHO_VAULT,
+                    TEAM_MULTISIG,
+                    false
+                );
+            }
+        }
+
+        {
+            // [TYPE 1] PAXG / USDC VAULT
+            address PAXG_USDC_VAULT = getVaultAddress(119);
+            {
+                VaultConfig memory VAULT_PAXG_USDC = VaultConfig({
+                    vault: PAXG_USDC_VAULT,
+                    vaultType: VAULT_TYPE.TYPE_1,
+                    supplyToken: PAXG_ADDRESS,
+                    borrowToken: USDC_ADDRESS,
+                    baseWithdrawalLimitInUSD: 5_000_000, // $5M
+                    baseBorrowLimitInUSD: 1_500_000, // $1.5M
+                    maxBorrowLimitInUSD: 1_500_000 // $1.5M
+                });
+
+                setVaultLimits(VAULT_PAXG_USDC); // TYPE_1 => 119
+
+                VAULT_FACTORY.setVaultAuth(
+                    PAXG_USDC_VAULT,
+                    TEAM_MULTISIG,
+                    false
+                );
+            }
+        }
+
+        {
+            // [TYPE 1] PAXG / USDT VAULT
+            address PAXG_USDT_VAULT = getVaultAddress(120);
+            {
+                VaultConfig memory VAULT_PAXG_USDT = VaultConfig({
+                    vault: PAXG_USDT_VAULT,
+                    vaultType: VAULT_TYPE.TYPE_1,
+                    supplyToken: PAXG_ADDRESS,
+                    borrowToken: USDT_ADDRESS,
+                    baseWithdrawalLimitInUSD: 5_000_000, // $5M
+                    baseBorrowLimitInUSD: 1_500_000, // $1.5M
+                    maxBorrowLimitInUSD: 1_500_000 // $1.5M
+                });
+
+                setVaultLimits(VAULT_PAXG_USDT); // TYPE_1 => 120
+
+                VAULT_FACTORY.setVaultAuth(
+                    PAXG_USDT_VAULT,
+                    TEAM_MULTISIG,
+                    false
+                );
+            }
+        }
+
+        {
+            // [TYPE 1] PAXG / GHO VAULT
+            address PAXG_GHO_VAULT = getVaultAddress(121);
+            {
+                VaultConfig memory VAULT_PAXG_GHO = VaultConfig({
+                    vault: PAXG_GHO_VAULT,
+                    vaultType: VAULT_TYPE.TYPE_1,
+                    supplyToken: PAXG_ADDRESS,
+                    borrowToken: GHO_ADDRESS,
+                    baseWithdrawalLimitInUSD: 1_000_000, // $1M
+                    baseBorrowLimitInUSD: 1_000_000, // $1M
+                    maxBorrowLimitInUSD: 2_500_000 // $2.5M
+                });
+
+                setVaultLimits(VAULT_PAXG_GHO); // TYPE_1 => 121
+
+                VAULT_FACTORY.setVaultAuth(
+                    PAXG_GHO_VAULT,
+                    TEAM_MULTISIG,
+                    false
+                );
+            }
+        }
+
+        {
+            // [TYPE 2] PAXG-XAUT<>USDC | smart collateral & normal debt
+            address PAXG_XAUT__USDC_VAULT = getVaultAddress(122);
+
+            {
```

Comment:
@bergben @xma8lu We also need increase the supply limits on dex for this vault?

Reply:
for all smart vaults, we can set the supply limits at the dex via team MS

Reply:
<img width="264" alt="Screenshot 2025-05-10 at 11 37 29 PM" src="https://github.com/user-attachments/assets/c471d7c5-e4a0-4683-840d-5fdff153adc6" />
hey i think it was set to 1.8M which is set here
![Screenshot 2025-05-10 at 11 37 49 PM](https://github.com/user-attachments/assets/722d3cf2-c32c-47cf-b0c9-baa7ac62876e)


i think we updated it to 2M let me update that in code shortly